packages/python/test/integration-1.test.ts

```diff
@@ -71,30 +71,11 @@ it('should match the probes against Python dev servers', async () => {
   }
 });
 
-const testsThatFailToBuild = new Map([
-  ['30-fail-build-invalid-pipfile', 'Unable to parse Pipfile.lock'],
-  [
-    '31-fail-build-invalid-python36',
-    'Python version "3.6" detected in Pipfile.lock is discontinued and must be upgraded.',
-  ],
-]);
+const allFixtures = fs.readdirSync(fixturesPath);
+const chunkFixtures = allFixtures.slice(0, 10);
```

Comment:
Why slice the first 10 fixtures?

Reply:
Oh, I see! `integration-2.test.ts` runs the rest.

Reply:
Yeah, this is not the best approach